spec.md

```diff
@@ -8,66 +8,96 @@ title: Semantic Specification
 
 
 ~~~
-Parentage/causal relationships for spans from a single trace
+Causal relationships for Spans in a single trace
 
-        [Span A]  <--(the root span)
+        [Span A]  ←←←(the root span)
             |
      +------+------+
      |             |
- [Span B]      [Span C]
+ [Span B]      [Span C] ←←←(Span A is Span C's parent)
      |             |
- [Span D]      +---+-----+
-               |         |
-           [Span E]  [Span F]
-                         |
-                +--------+--------+
-                |        |        |
-            [Span G] [Span H] [Span I]
+ [Span D]      +---+-------+
+               |           |
+           [Span E]    [Span F] >>> [Span G] >>> [Span H]
```

Comment:
@adriancole @xylus the concept "sequence predecessors" ended up evolving (in this PR discussion and in our OT hangout today with @yurishkuro and @dkuebric) into something more like `Follows()` per @tinkerware's comment here: https://github.com/opentracing/opentracing.github.io/pull/100#issuecomment-229404325

That whole thread is pretty interesting IMO.